problemsets-mat223/modules/module5.tex

```diff
@@ -53,7 +60,22 @@
 	and let $\vec v=\mat{-1\\-1}$. Use the fact that $\vec v-\Proj_\ell\vec v$ is a normal vector to $\ell$
 	to find Find $\Proj_\ell \vec v$.
 
-	XXX Finish
+	Since $\vec v-\Proj_\ell\vec v$ is a normal vector to $\ell$, $\vec v-\Proj_\ell\vec v$ is orthogonal to $\vec d=\mat{1\\1}$. Thus, let $\Proj_\ell\vec v = \mat{x\\y}$ for some unknown $x,y\in\R$, we have
```

Comment:
`, $\vec v-\P` -> `, we know $\vec v-\P`   I always try to put an English word after a `,` so it doesn't look like too much math in a row.

Break this up into shorter lines (i.e., add some newlines to the TeX code)